webapp/src/client.ts

```diff
@@ -31,7 +34,7 @@ export default class CallsClient extends EventEmitter {
     private audioTrack: MediaStreamTrack | null;
     public isHandRaised: boolean;
 
-    constructor() {
+    constructor(getState: GetStateFunc) {
```

Comment:
I am not that in love with this kind of coupling if it can be avoided. Could we pass the ice servers directly instead?

Reply:
Sure, passing the iceServers is cleaner, thanks